Mend CLI/github-action_container_scan.yml

```diff
@@ -8,10 +8,10 @@ jobs:
   mend-cli:
     name: Mend-CLI
     env:
-      MEND_EMAIL: ${{secrets.EMAIL}}
-      MEND_USER_KEY: ${{secrets.USERKEY}}
-      MEND_URL: https://saas.whitesourcesoftware.com
-      MEND_ORG_UUID: ${{secrets.SCAORG}}
+      MEND_EMAIL: ${{secrets.MEND_EMAIL}}
+      MEND_USER_KEY: ${{secrets.MEND_USERKEY}}
+      MEND_URL: https://saas.mend.io
+      MEND_ORG_UUID: ${{secrets.MEND_SCA_ORG}}
     runs-on: ubuntu-latest
     steps:
```

Comment:
With our current GH and other examples, we don't split the download of the scanner with the execution, I think we should merge it to one step and use `echo`.
Also, Change the name to: `Mend Container Scan`